### PR TITLE
Polymorphic objects in API

### PR DESCRIPTION
**Pull request by mkurek**
Return polymorphic objects to API using dedicated serializers for each inheriting model.
Use this polymorphic logic to BaseObject in API.
Added `polymorphic_select_related` and `polymorphic_prefetch_related` to Polymorphic queryset.
Minor performance improvements in polymorphic logic.

Connected to #1789

### CODE REVIEW

File: src/ralph/api/serializers.py [hunk 1]
```diff
@@ -143,8 +143,22 @@ def build_field(self, *args, **kwargs):
...
 
+serializers_registry = {}
+
+
+class RalphAPISerializerMetaclass(serializers.SerializerMetaclass):
+    def __new__(cls, name, bases, attrs):
+        attrs['_serializers_registry'] = serializers_registry
+        new_cls = super().__new__(cls, name, bases, attrs)
+        meta = getattr(new_cls, 'Meta', None)
+        if getattr(meta, 'model', None):
```
Comment (ar4s) [inline src/ralph/api/serializers.py:154]: `if meta and getattr(meta, 'model', None):` to prevent `AttributeError`

Comment (mkurek) [inline src/ralph/api/serializers.py:154]: getattr is working fine with None:

```
>>> x = None
>>> print(getattr(x, 'model', None))
None
```

**Comment by ar4s** [comment]
:ok:

**Comment by szok** [comment]
:ok:

**Pull request merged**